packages/mdc-snackbar/util.ts

```diff
@@ -77,8 +76,12 @@ function announce(ariaEl, labelEl = ariaEl) {
   labelEl.setAttribute(ARIA_LIVE_LABEL_TEXT_ATTR, labelText);
 
   setTimeout(() => {
-    // Allow screen readers to announce changes to the DOM again.
-    ariaEl.setAttribute('aria-live', priority);
+    if (priority === null) {
```

Comment:
Is this fixing a bug at the same time as the conversion? We should probably carry this over to master so it gets noted...

Reply:
Discussed w/ Andy; it's technically fixing a bug, but for an edge case that won't occur in valid usage of the component. This is probably fine to just fix in the TS branch.

Reply:
Nope, it's just to make the TypeScript compiler happy.
`getAttribute()` can technically return `null`.
It's not a use case we support, however; the `aria-live` attribute should _always_ be present.